clients/privacy-center/config/config.json

```diff
@@ -71,7 +71,7 @@
           "url": "https://example.com/privacy#analytics",
           "default": true,
           "highlight": false,
-          "cookieKeys": ["tracking"],
+          "cookieKeys": ["analytics"],
```

Comment:
This is unrelated, but I felt it was wrong to group email marketing & product analytics together in a single consent preference 👍